Cycle_Sort/Cycle_Sort.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var x int
+    fmt.Printf("Enter the size of the array : ")
+    fmt.Scan(&x)
+    fmt.Printf("Enter the elements of the array : ")
+    a := make([]int, x)
+    for i := 0; i < x; i++ {
+        fmt.Scan(&a[i])
+    }
+    arr := a    // A copy of the array 'a' is assigned to 'randomSlice' 
+	CycleSort(&arr)
+	fmt.Printf("After sort: %v\n", arr)
+}
+
+func CycleSort(arr *[]int) int {
+	writes := 0
```

Comment:
Looks like there are more than 4 spaces indentation in this function. Please check.